### ISSUE TITLE
Could not connect to  nodes

### ISSUE THREAD

**Issue by pauluk333** [ethereum/mist#3721]
- [x] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

no luck

```
Version: `0.0.0`
OS & Version: windows/linux/osx
Node version: `geth 0.0.0` 
Number of blocks synchronized: 0
```
windows version, latest version, cant get node version as wont allow me in , cant get block synchronized either 

[error.docx](https://github.com/ethereum/mist/files/1775644/error.docx)

**Comment by simon66** [comment]
@pauluk333 
No need to attach a word doc, you can attach in images too.
Here's the picture of pauluk333's issue
![a](https://image.ibb.co/ih7Bhn/capture.png)

**Comment by simon66** [comment]
Hi @pauluk333 
Which version of `Mist` are you using?

**Comment by evertonfraga** [comment]
The connection has probably timed out while trying to get some peers. 

Please try again, making sure you don't have any network configurations preventing it from connecting to other peers.

To make a proper test, you can start Geth separately: https://github.com/ethereum/mist/wiki#start-the-node-manually

**Comment by ryanio** [comment]
Hey, we just released a new version (`0.11.0`) that has new syncing features - please give it a try! If you have any new problems that you can reproduce, please create a new issue with the details on how 
we can help. Thanks!
